src/razor/razor.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
The logic in this file uses `vscode` APIs directly, not going through `vscodeAdapter`, for reasons we've discussed previously. The `microsoft.aspnetcore.razor.vscode` NPM package uses `vscode` directly extensively, so nothing extra would be gained by putting this tiny bit of wrapper logic through additional abstraction.

Later on we do want to move `microsoft.aspnetcore.razor.vscode` entirely to share OmniSharp-VSCode's `vscodeAdapter` pattern for testability, but that's a bit further out.